### PR TITLE
Regenerate conjure

### PR DESCRIPTION
**Pull request by ericanderson**
- Makes lint-staged ugly on commit (but not fail vscode)
- Upgrades to latest conjure-lite
- Fixes/improves scripts, including resetting files that have only changed by copyright year.
- Regenerates.

No changelog, its not user facing.

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged
+TERM=dumb npx lint-staged
```
Comment (ssanjay1) [inline .husky/pre-commit:4]: I may not have searched google hard enough, but it seems like this just disables color output right? Does that really choke on big changes?

File: scripts/generateConjure.sh [hunk 2]
```diff
@@ -25,13 +25,33 @@ function generateConjure() {
...
 generateConjure "com.palantir.ontology" "ontology-metadata-api" "${SCRIPT_DIR}/../packages/client"
+
+# Reset git changes if the generated files are only changed by copyright year
+git status --porcelain --untracked-files=no | while read line; do
+    status=$(echo "$line" | awk '{print $1 }')
+    file=$(echo "$line" | awk '{print $2 }')
+
+    if [[ "$status" == "D" ]]; then
+        # do nothing
+        bnitgub="da"
```
Comment (ssanjay1) [inline scripts/generateConjure.sh:44]: where do you come up with these names lol

File: scripts/generateConjure.sh [hunk 3]
```diff
@@ -25,13 +25,33 @@ function generateConjure() {
...
+    else 
+        v=$(git diff -U0 $file | tail +5 | md5sum)
+        expected="ce0376e23ce2aa3d0dcd56c2befc2993  -"
```
Comment (ssanjay1) [inline scripts/generateConjure.sh:47]: So this is the hash we'd expect if only the copyright year changes?

**Review by ssanjay1 — APPROVED** [review]

File: scripts/generateConjure.sh (hunk 3 above)
Comment (ericanderson) [reply scripts/generateConjure.sh:47]: Yes

File: scripts/generateConjure.sh (hunk 2 above)
Comment (ericanderson) [reply scripts/generateConjure.sh:44]: I needed something for the if statement so bash wouldnt be mad.

File: .husky/pre-commit (hunk 1 above)
Comment (ericanderson) [reply .husky/pre-commit:4]: It disables the ansi control code which seems to upset vscode

**Pull request merged**